package.json

```diff
@@ -48,7 +48,8 @@
     "test-markup": "mocha test/markup",
     "test-detect": "mocha test/detect",
     "test-browser": "mocha test/browser",
-    "test-parser": "mocha test/parser"
+    "test-parser": "mocha test/parser",
+    "use-browser-tools": "rm -rf ./build && npm run build_and_test_browser && http-server ./tools"
```

Comment:
Wait, this is for tools? Why?  You can just `open tools/developer.html` locally, it doesn't require a web server at all to work...